### ISSUE TITLE
Lua error when I applie a CoA to an enemy target

### ISSUE THREAD

**Issue by Splaser** [rgd87/ClassicAuraDurations#5]
I got an "application is nil " error when I casted CoA.
![1](https://user-images.githubusercontent.com/9554026/63829018-e1166a80-c99a-11e9-940d-724203fc215b.png)

**Comment by d87** [comment]
Can you give more info about when and how often it happens?

**Comment by Splaser** [comment]
![1](https://user-images.githubusercontent.com/9554026/63836762-be8d4d00-c9ac-11e9-80e0-53dfa83e908c.png)
Sorry. You may check this one.

**Comment by d87** [comment]
It should be fixed in the latest update, but i still have no idea what is exacly going on, it seems to be something specific to chinese localization

**Comment by Splaser** [comment]
Yeah. It only happens when I cast CoA on any target. Other spells are working well.

**Comment by d87** [comment]
https://github.com/rgd87/ClassicAuraDurations/releases
Install 1.13.22 and see if it helps without breaking CoA timer in the process.

**Comment by Splaser** [comment]
OK. Thanks. I will try it soon after a 2000 waiting queue.

**Comment by Splaser** [comment]
![1](https://user-images.githubusercontent.com/9554026/63946713-24123400-caa8-11e9-9deb-93c3c7f54951.png)
There is no error this time. But it seems the CoA timer is still broken.

**Comment by d87** [comment]
I don't know then, get some Chinese guy to figure it out and tell me
I tried running Chinese client in sandbox mode and everything seems fine on the side of spellName to spellID translation table.
So It's probably something weird in the combat log, maybe Chinese client has different spell name in combat log for CoA than it is in the spell book.

**Comment by Splaser** [comment]
My client is a Simplefied Chinese version client. Not the Traditional Chinese one which might work fine.

**Comment by d87** [comment]
I tried both of them in sandbox.
Also i'll leave a guide how to diagnose combat log events in case anyone wants to do it
1) Install NugRunning for classic https://github.com/rgd87/NugRunning/releases
2) run /nrun debug
3) it should print "0 痛苦诅咒 ... " for CoA events

**Comment by EKE00372** [comment]
problem happened because this:
https://cn.classic.wowhead.com/search?q=%E7%97%9B%E8%8B%A6%E8%AF%85%E5%92%92#uncategorized-spells

Fire Vulnerability translate as "痛苦诅咒" in zhCN.
maybe can use icon texture to make a filter?

**Comment by d87** [comment]
Oh thank god... Arctium got broken with recent update so i couldn't even get into sandbox anymore.
Combat log only tells spell name, so icons are not an option.
But i'll just make so that warlocks will see CoA and mages Fire Vulnerability, but not both

**Comment by EKE00372** [comment]
This actually a wrong translate, I've report this to blizzard, but don't know when will be firxed.....

**Comment by Splaser** [comment]
Why did blizz make combatlog without any spellid?  Really a backforward.

**Comment by d87** [comment]
Intentionally to mess up addons, it was fine in beta but then they removed spell ids in august stress test.

**Comment by wolfcon** [comment]
> Intentionally to mess up addons, it was fine in beta but then they removed spell ids in august stress test.

For original taste of WoW. 🤪 Block many functions and APIs
